src/test/java/com/endava/internship/mocking/service/PaymentServiceTest.java

```diff
@@ -40,27 +47,35 @@ void setUp() {
         paymentRepository = Mockito.mock(InMemPaymentRepository.class);
         paymentService = new PaymentService(userRepository,paymentRepository,basicValidationService);
     }
+    @AfterEach
+    void tearDown() {
+        Mockito.verifyNoMoreInteractions(basicValidationService);
+        Mockito.verifyNoMoreInteractions(userRepository);
+        Mockito.verifyNoMoreInteractions(paymentRepository);
+
```

Comment:
blank line also use
CTRL + ALT + L for all file to format

Reply:
fixed